gradle-examples/ejb-jaxrs-mail/gradle.properties

```diff
@@ -0,0 +1,5 @@
+#
+# Copyright (c) 2018, PayPal. All Rights Reserved.
+#
```

Comment:
This looks weird :-) But I'm not a lawyer.

Reply:
Oops! My true identity had been found!! :)

Let me fix that today.